src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisPerformanceReportJobConfigurationTest.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.ndmis.performance
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.batch.core.ExitStatus
+import org.springframework.batch.core.Job
+import org.springframework.batch.core.JobExecution
+import org.springframework.batch.core.JobParametersBuilder
+import org.springframework.batch.test.JobLauncherTestUtils
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.TimestampIncrementer
+import kotlin.io.path.createTempDirectory
+import kotlin.io.path.fileSize
+import kotlin.io.path.listDirectoryEntries
+import kotlin.io.path.name
+import kotlin.io.path.pathString
+
+@Component
+class NdmisPerformanceJobLauncherTestUtils : JobLauncherTestUtils() {
+  @Autowired
+  override fun setJob(@Qualifier("ndmisPerformanceReportJob") job: Job) = super.setJob(job)
+}
```

Comment:
This was frustrating. The only way I could get a specific job passed down to `JobLauncherTestUtils` if

1. Subclassed `JobLauncherTestUtils` (from the `org.springframework.batch:spring-batch-test` library)
2. Added a `@Qualifier` to `override fun setJob(...)`